app/scripts/vueCore.js

```diff
@@ -30,6 +30,7 @@
             charts: {},
             groupCount: 0,
             updateSpecialCharts: false,
+            showShareButton: true,
             showSaveButton: true,
```

Comment:
Disable this.

Reply:
I thought the default value of showShareButton is true and we can manipulate it index.html by setting iViz.vue.manage.getInstance().showShareButton = true/false.

Reply:
Let's change the default of saving to false.